deployment/deployment.json

```diff
@@ -3721,6 +3721,36 @@
       }
     }
   },
+  "sushiswap-v3": {
+    "schema": "dex-amm",
+    "base": "uniswap-v3-forks",
+    "protocol": "sushiswap-v3",
+    "project": "sushiswap",
+    "deployments": {
+      "sushiswap-v3-ethereum": {
```

Comment:
I just realised I didn't add all of them here, assume I should? @dmelotik

Reply:
Hi Ola! Yes, please add all of the networks to the deployment.json.

Reply:
https://github.com/messari/subgraphs/pull/2151/commits/a282811d6d3f0d347f2b9dad66de08189230888f